pkg/collateral/control.go

```diff
@@ -56,6 +76,144 @@ type Control struct {
 	ManPageInfo doc.GenManHeader
 }
 
+// Constants used in zsh completion file
+const (
+	zsh_initialization = `#compdef istioctl
+
+__istioctl_bash_source() {
```

Comment:
pkg/collateral is a generic component used by all our binaries. We should not have istioctl-specific code in here.

Reply:
I moved to constants down to within the c.EmitZshCompletion(), made them _istio_ wide, and updated to use the command name within the zshInitialization so can be used as part of any command.